src/rprblender/properties/render.py

```diff
@@ -419,18 +419,17 @@ def is_tile_render_available(self):
     )
 
     render_quality_items = [
-        ('FULL', "Full", "Full render quality. If using CPU this is the only mode available")
+        ('FULL', "Legacy", "Full render quality using RPR 1.")
     ]
     if pyhybrid.enabled:
         render_quality_items += [
             ('HIGH', "High", "High render quality"),
             ('MEDIUM', "Medium", "Medium render quality"),
             ('LOW', "Low", "Low render quality"),
         ]
-    if pyrpr2.enabled:
-        render_quality_items += [
-            ('FULL2', "RPR 2.0 (Beta)", "Full render quality with RPR 2 (Beta)")
-        ]
+    render_quality_items += [
+        ('FULL2', "Full", "Full render quality using RPR 2, including hardware ray tracing support.")
```

Comment:
In UI 'Full' quality is last item, better to have it as first. Would be good to move this item to 422 line.